tensorflow/python/ops/math_ops.py

```diff
@@ -3230,6 +3295,11 @@ def log_sigmoid(x, name=None):
 
   Specifically, `y = log(1 / (1 + exp(-x)))`.  For numerical stability,
   we use `y = -tf.nn.softplus(-x)`.
+  
+  For example:
+  >>> x = 32.0
+  >>> tf.math.log_sigmoid(x)
+  <tf.Tensor: shape=(), dtype=float32, numpy=-1.2664166e-14>
```

Comment:
Can you use a smaller value than 32, so the doctest doesn't need so much precision?